bundles/org.openhab.binding.vesync/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -12,6 +12,7 @@
 			<property name="Registration Time"/>
 			<property name="Country Code"/>
 			<property name="Accept Language"/>
+			<property name="thingTypeVersion">1</property>
```

Comment:
the ve sync bridge is not upgraded, so this can be removed
```suggestion
```

Reply:
Thank you - I had added it to the wrong thing, as you spotted here and in the other comment.